src/data/Roster.cpp

```diff
@@ -64,15 +70,16 @@ void Roster::reset(){
 // get a vector of consecutive states that will result from applying the
 // the roster from a given initial state
 //
-vector<State> Roster::getStates(const State& stateIni) {
+vector<State> Roster::getStates(const State& stateIni, Scenario* pScenario) {
```

Comment:
Why an attribute scenario, since there already is a member variable pScenario_?

Reply:
No, it's not the case for the object roster.

Reply:
I don't think it is.

Reply:
Line 67 of Roster.h :`const Scenario* pScenario_; `
And this variable is actually used in `Roster::getStates()`, on line 82 : 
`nextState.addDayToState(states[day], shiftType, shifts_[day], pScenario_->timeDurationToWork_[shifts_[day]]);`